casbin/model/policy.py

```diff
@@ -21,6 +21,19 @@ def print_policy(self):
             for key, ast in self.model[sec].items():
                 log.log_print(key, ": ", ast.value, ": ", ast.policy)
 
+    def remove_policy(self, sec, ptype, rule):
+        """remove a policy rule to the model."""
+        if sec not in self.model.keys():
+            return False
+        if ptype not in self.model[sec]:
+            return False
+
+        for i, r in enumerate(self.model[sec][ptype].policy):
+            if rule == r:
+                self.model[sec][ptype].policy.pop(i)
+                return True
```

Comment:
Hi @niceyida , please put this function right after ``addPolicy()`` instead of here.